### PR TITLE
adds a sliceutils method to flatten a slice of slice of type I to a s…

### PR DESCRIPTION
**Pull request by vhoen**
…ingle level slice of type I

### CODE REVIEW

**Review by Goldziher — APPROVED** [review]

**Comment by Goldziher** [comment]
Great, you can merge. I sent an invite for you to have access to this repository - so you should be able to merge now 😉

**Comment by vhoen** [comment]
Thanks, I was waiting for approval

**Pull request merged**
